packages/lodestar/src/chain/chain.ts

```diff
@@ -176,27 +168,46 @@ export class BeaconChain extends (EventEmitter as {new (): ChainEventEmitter}) i
     epochCtx.loadState(state);
     await this.db.stateCache.add({state, epochCtx});
     this.logger.info("Chain started, waiting blocks and attestations");
-    this.clock = new LocalClock(this.config, state.genesisTime);
-    await this.clock.start();
-    await this.forkChoice.start(state.genesisTime, this.clock);
+    this.clock = new LocalClock({
+      config: this.config,
+      emitter: this.emitter,
+      genesisTime: state.genesisTime,
+      signal: this.abortController.signal,
```

Comment:
why not create separate abort controller inside clock?

Reply:
:shrug: It is nice to be able to control from an outside source (that may not even be in scope here).

If we want to make the `IBeaconClock` interface more flexible, we can include a public `abort` method, make `signal` an optional input to `LocalClock`.

Reply:
I liked that we had start/stop standard everywhere but ok